modules/imports/src/main/scala/org/dhallj/imports/CorsComplianceCheck.scala

```diff
@@ -4,7 +4,6 @@ import java.net.URI
 
 import cats.effect.Sync
 import org.dhallj.core.DhallException.ResolutionFailure
-import org.dhallj.imports.ResolveImportsVisitor._
```

Comment:
Thanks for catching this. We should probably set up the unused import warnings, etc.